lib/src/map/options/enabled_gestures.dart

```diff
@@ -0,0 +1,287 @@
+import 'package:flutter_map/flutter_map.dart';
+import 'package:meta/meta.dart';
+
+@immutable
+class EnabledGestures {
+  /// Use this constructor if you want to set all gestures manually.
+  ///
+  /// Prefer to use the constructors [InteractiveFlags.all] or
+  /// [InteractiveFlags.none] to enable or disable all gestures by default.
+  ///
+  /// If you want to define your enabled gestures using bitfield operations,
+  /// use [InteractiveFlags.bitfield] instead.
+  const EnabledGestures({
+    required this.drag,
+    required this.flingAnimation,
+    required this.twoFingerMove,
+    required this.twoFingerZoom,
+    required this.doubleTapZoomIn,
+    required this.doubleTapDragZoom,
+    required this.scrollWheelZoom,
+    required this.twoFingerRotate,
+    required this.keyTriggerDragRotate,
+    required this.trackpadZoom,
+  });
+
+  /// Shortcut constructor to allow all gestures that don't rotate the map.
+  const EnabledGestures.noRotation()
```

Comment:
Personally, I would remove this constructor, as it's now easy to do with `byGroup` (especially if they have optional parameters). WDYT?

Reply:
Having no map rotation is probably one of the most used map options. Making it as easy as can be therefore makes sense to me.
The additional constructor doesn't has any performance or mentionable maintainence penalty.